salt/reboot/init.sls

```diff
@@ -0,0 +1,42 @@
+##################################################
+# Configuration for the reboot manager
+##################################################
+
+# `max_holders` contains the maximum number of lock holders for the cluster.
+{% set max_holders = salt['pillar.get']('etcd:masters', '1')|int %}
+{% if max_holders > 1 %}
+  {% set max_holders = max_holders - 1 %}
```

Comment:
Fixed. For the base case we'll set `max_holders: 1`. For the general case we are going for `max_holders // 2`, which should give us the right number.